### ISSUE TITLE
Developing a framework for robust research, monitoring and restoration activities

### ISSUE THREAD

**Issue by aebudden** [NCEAS/oss-2017#32]
Author: **Edward Sherwood**
Topics: Restoration

## Background
The ecosystem value of the Gulf of Mexico (GOM) and its economic outputs relative to investments in GOM research, monitoring and restoration activities pale in comparison to other, similarly large ecosystems that are also considered significant national assets (e.g. Chesapeake Bay, Pacific northwest, Great Lakes, etc.). The Deepwater Horizon oil spill highlighted that considerable knowledge gaps exist within the GOM ecosystem ranging from finer- (e.g. extent and condition of sensitive/critical habitats and biota) to larger-scale issues (e.g. GOM circulation and pollution trajectories). The current and future input of significant funds through the RESTORE Act and NRDA processes will provide much needed resources to research, monitoring and restoration activities that will hopefully fill existing knowledge gaps within the GOM. However, a lack of coordinated effort to track the implementation and outcomes of these varied activities in the GOM currently exists. As a regionally-specific GOM researcher, my agency and I have worked as GOM restoration project concept reviewers, collaborators and direct implementers. I recognize my research and my agencies activities are only one small component of a much larger and broader GOM ecosystem-wide restoration effort. However, what is sorely lacking today is a mechanism to effectively guide future activities based on the cumulative body of research, monitoring and restoration activities currently being investigated through various funding mechanisms in the GOM.

## Synthesis summary
My proposed group synthesis project will attempt to fill this existing knowledge gap. First, I propose to integrate various GOM-wide physical and biological datasets to better ascertain current threats and future risks to various GOM ecosystem components at distinct estuary, regional coast, and GOM-wide scales. Second, I propose to mine the monitoring outcomes and lessons-learned from the vast amount of funded GOM research/restoration projects and overlay this information on the integrated ecosystem resource dataset developed in the first step. Third, I propose to develop an assessment approach that identifies the most cost-effective future restoration activities that takes into account current and future ecosystem threats, as well as, the cumulative knowledge and lessons-learned from previously funded activities. The resulting synthesis project outcomes will be a more informed decision support tool for future research, monitoring and restoration activities at various GOM scales that takes into account the (restoration) costs relative to (ecosystem and economic) benefits.

## Significance and Impact
I believe this project concept would directly address Goals 2 & 3 of the Gulf Research Program, as the fundamental connections of GOM ecosystems and the economic and ecosystem benefits of its continued restoration will continue to be refined and explored through the process outlined above. In the end, the proposed group synthesis project aims to develop more robust research, monitoring and restoration activities within the GOM and steer limited resources towards projects that address the most critical ecosystem (and resulting economic) threats for this important system.

**Comment by ascyoung** [comment]
While this a very ambitious proposal, I feel that the focus on a Gulf-wide decision support tool is a great approach. The agency I currently work for (NOAA RESTORE Act Science Program) recently completed a funding opportunity that had a decision support tool focus. We received ~50 DST proposals but none of them were able to approach the DST from a Gulf-wide perspective.  Most of the DSTs (many proposals were an expansion on an existing DST) were quite local, at best regional. That said, there are definitely lessons we can learn from these existing DSTs, particularly since many are run by multi-disciplinary teams that include both academics and resource managers.

**Comment by fawda123** [comment]
@esherwoo77 Seems like the concepts here could be merged with some of my ideas in #28. What's lacking in my proposal is a clear goal beyond synthesizing the data, whereas this is the first objective of yours.  I really like the idea of using the dataset as a decision-support tool, particularly as a threat/stressor categorization tool.

**Comment by esherwoo77** [comment]
@ascyoung Agreed, the growing body of research being funded through the RESTORE Act will definitely be useful to broaden and improve our restoration activities. Our program has worked to some extent with local NOAA RESTORE contacts (e.g. Steve Giordano) to ensure that lessons learned from funded research/monitoring are quickly disseminated. If you know of any data portals that could be shared on previously-funded research/monitoring outcomes in the Gulf, that would be very helpful for my synthesis project concept (as well as my program's restoration efforts in the region ;-).

**Comment by JessicaHenkel** [comment]
I agree with @fawda123 and @esherwoo77 that  there is a lot of potential for merging this proposal (#32) with #28, #34 and potentially #22. I think focusing on one benefit (e.g. Water Quality) could make a synthesis project more feasible.

**Comment by pvarelag** [comment]
I agree, this project can be implemented by using the methodology proposed in #22.
